<launch>
	<node name="color_seg" pkg="tum_os" type="color_seg" args="1" respawn="true" output="screen"/>
  <node name="collect_data" pkg="tum_os" type="collect_data" respawn="true" output="screen"/>
	<node name="pick_place" pkg="tum_os" type="pp_try" respawn="false" output="screen"/>
</launch>
